docs/nddata/ccddata.rst

```diff
@@ -0,0 +1,198 @@
+:orphan:
+
+.. _ccddata:
+
+CCDData class
+=============
+
+Getting started
+---------------
+
+Getting data in
++++++++++++++++
+
+Creating a `~astropy.nddata.CCDData` object from any array-like data is easy:
+
+    >>> import numpy as np
+    >>> from astropy.nddata import CCDData
+    >>> ccd = CCDData(np.arange(10), unit="adu")
+
+Note that behind the scenes, this creates references to (not copies of) your
+data when possible, so modifying the data in ``ccd`` will modify the
+underlying data.
+
+You are **required** to provide a unit for your data. The most frequently used
+units for these objects are likely to be ``adu``, ``photon`` and ``electron``, which
+can be set either by providing the string name of the unit (as in the example
+above) or from unit objects:
+
+    >>> from astropy import units as u
+    >>> ccd_photon = CCDData([1, 2, 3], unit=u.photon)
+    >>> ccd_electron = CCDData([1, 2, 3], unit="electron")
+
+If you prefer *not* to use the unit functionality then use the special unit
+``u.dimensionless_unscaled`` when you create your `~astropy.nddata.CCDData`
+images:
+
+    >>> ccd_unitless = CCDData(np.zeros((10, 10)),
+    ...                        unit=u.dimensionless_unscaled)
+
+A `~astropy.nddata.CCDData` object can also be initialized from a FITS file:
+
+    >>> ccd = CCDData.read('my_file.fits', unit="adu")  # doctest: +SKIP
+
+If there is a unit in the FITS file (in the ``BUNIT`` keyword), that will be
+used, but a unit explicitly provided in ``read`` will override any unit in the
```

Comment:
that sounds a bit weird. 

Maybe better to use `but explicitly providing a unit in ...`